packages/meditrak-server/src/routes/mapOverlayGroupRelations/GETMapOverlayGroupRelations.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { GETHandler } from '../GETHandler';
+import { assertAnyPermissions, assertBESAdminAccess } from '../../permissions';
+import {
+  assertMapOverlayGroupRelationsGetPermissions,
+  createMapOverlayGroupRelationDBFilter,
+  createRelationsViaParentOverlayGroupDBFilter,
+} from './assertMapOverlayGroupRelationsPermissions';
+import { assertMapOverlayGroupsGetPermissions } from '../mapOverlayGroups';
+
+/**
+ * Handles endpoints:
+ * - /mapOverlayGroupRelations
+ * - /mapOverlayGroupRelations/:mapOverlayGroupRelationId
+ */
+export class GETMapOverlayGroupRelations extends GETHandler {
+  permissionsFilteredInternally = true;
+
+  customJoinConditions = {
+    map_overlay_group: ['map_overlay_group.id', 'map_overlay_group_relation.map_overlay_group_id'],
+  };
+
+  async findSingleRecord(mapOverlayId, options) {
+    const mapOverlay = await super.findSingleRecord(mapOverlayId, options);
+
+    const mapOverlayChecker = accessPolicy =>
+      assertMapOverlayGroupRelationsGetPermissions(accessPolicy, this.models, mapOverlayId);
```

Comment:
I'm not sure if this will work: `assertMapOverlayGroupRelationsGetPermissions` expects a map overlay group id, but if I'm not mistaken here we have access to a `mapOverlayGroupRelationId` (named `mapOverlayId` in the code above)